wd_helper.au3

```diff
@@ -980,12 +980,15 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ElementSelectAction
 ; Description ...: Perform action on desginated Select element.
-; Syntax ........: _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
+; Syntax ........: _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand[, $vParameter = Null])
 ; Parameters ....: $sSession       - Session ID from _WD_CreateSession
 ;                  $sSelectElement - Element ID of Select element from _WD_FindElement
 ;                  $sCommand       - Action to be performed. Can be one of the following:
-;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's options
-;                  |VALUE   - Retrieve current value
+;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's Options
+;                  |SELECTEDINDEX  - Retrieve 0-based index of selected option, from Options collection of designated Select element. Special value -1 is used to deselect options
```

Comment:
This is fairly wordy. IMO this part isn't needed --

`, from Options collection of designated Select element`

Also couldn't the following go under `remarks` instead?

`If no option is selected, selectedIndex will return -1`

Reply:
[descriptions and comments](https://github.com/Danp2/WebDriver/pull/175/commits/8cc24bcf0d97ff0c790cda72f60c1a8728369c67)